share/tools/web_config/sample_prompts/sorin.fish

```diff
@@ -0,0 +1,60 @@
+# Name:   Sorin
+# Author: Ivan Tham <ivanthamjunhoe@gmail.com>
+
+# If login shell, don't do anything
+if status --is-login; and function fish_prompt; echo -n '% '; end; end
```

Comment:
It prevent errors for unsupportable characters in the linux framebuffer (**Ctrl+Alt+F2**).